tasks/task962_ancora-ca-ner_missing_word_prediction.json

```diff
@@ -928,13 +932,18 @@
             ]
         },
         {
-            "input": "El Grup Municipal d' IC-V-LP presentarà una moció al Ple ____ que s' adeqüi les platges de Tarragona a les persones minusvàlides .",
+            "input": "El Grup Municipal d' IC-V-LP presentar\u00e0 una moci\u00f3 al Ple ____ que s' adeq\u00fci les platges de Tarragona a les persones minusv\u00e0lides .",
             "output": [
                 "demanant"
             ]
         }
     ],
-    "Categories" : ["Sentence Generation"],
-    "Contributors": ["Surya Singulur"]
-
+    "Categories": [
+        "Question Answering -> Fill in the Blank",
+        "Question Answering -> Contextual Question Answering -> Missing Knowledge",
```

Comment:
Missing Knowledge means we have a context (i.e., passage) and a question based on the passage, but the answer to the question is not present in the passage and we can use the passage to get clues for searching the answer. In the fil-in-the-blank tasks we don't have passage, so they are not Missing Knowledge.